### DISCUSSION TITLE
after prometheus installation got error at frontend

### DISCUSSION THREAD

**Discussion by devilathag** [learndevopsonline/learndevopsonline#1518] [Q&A]
Dear Team,

After prometheus installation i have run the below command in frontend but it's getting error

 ansible-pull -i localhost, -U https://github.com/devilathag/Infra-Ansible1 main.yml -e role_name=frontend
pls check and do the needful

<img width="953" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/144150880/13091ac2-dd33-4a78-a557-718bd29de5f1">

**Comment by sreedharm07** [comment]
@devilathag 
your machine is inactive to check, anyway please check parameters are correctly created or not , policy.

**Comment by devilathag** [comment]
Ok, I will check.
